modules/ducktests/tests/ignitetest/services/ignite_app.py

```diff
@@ -109,5 +108,4 @@ def extract_results(self, name):
         return res
 
     def update_config_with_globals(self):
-        if self.globals.get("use_ssl", False):
-            self._update_ssl_config_with_globals("client", DEFAULT_CLIENT_KEYSTORE)
+        self._update_ssl_config_with_globals("client")
```

Comment:
Why "client"?

Reply:
Because currently it's done like that. I just modify function parsing this parameter from globals.
May be this parameter should be changed in different task.